packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -169,12 +225,77 @@ function GridFilterForm(props: GridFilterFormProps) {
     [currentOperator],
   );
 
+  const FilterFormDeleteIcon = React.useMemo(
+    () =>
+      styled(rootProps.components.BaseFormControl, {
+        name: 'MuiDataGrid',
+        slot: 'FilterFormDeleteIcon',
+        overridesResolver: (_, styles) => styles.filterFormDeleteIcon,
+      })({
+        flexShrink: 0,
+        justifyContent: 'flex-end',
+        marginRight: 0.5,
+        marginBottom: 0.2,
+      }),
+    [rootProps.components.BaseFormControl],
+  );
+
+  const FilterFormLinkOperatorInput = React.useMemo(
+    () =>
+      styled(rootProps.components.BaseFormControl, {
+        name: 'MuiDataGrid',
+        slot: 'FilterFormLinkOperatorInput',
+        overridesResolver: (_, styles) => styles.filterFormLinkOperatorInput,
+      })({
+        minWidth: 60,
+        display: hasLinkOperatorColumn ? 'block' : 'none',
+        visibility: showMultiFilterOperators ? 'visible' : 'hidden',
+      }),
+    [rootProps.components.BaseFormControl, hasLinkOperatorColumn, showMultiFilterOperators],
+  );
+
+  const FilterFormColumnInput = React.useMemo(
+    () =>
+      styled(rootProps.components.BaseFormControl, {
+        name: 'MuiDataGrid',
+        slot: 'FilterFormColumnInput',
+        overridesResolver: (_, styles) => styles.filterFormColumnInput,
+      })({ width: 150 }),
+    [rootProps.components.BaseFormControl],
+  );
+
+  const FilterFormOperatorInput = React.useMemo(
+    () =>
+      styled(rootProps.components.BaseFormControl, {
+        name: 'MuiDataGrid',
+        slot: 'FilterFormOperatorInput',
+        overridesResolver: (_, styles) => styles.filterFormOperatorInput,
+      })({ width: 120 }),
+    [rootProps.components.BaseFormControl],
+  );
+
+  const FilterFormValueInput = React.useMemo(
+    () =>
+      styled(rootProps.components.BaseFormControl, {
+        name: 'MuiDataGrid',
+        slot: 'FilterFormValueInput',
+        overridesResolver: (_, styles) => styles.filterFormValueInput,
+      })({ width: 190 }),
+    [rootProps.components.BaseFormControl],
+  );
+
   return (
     <GridFilterFormRoot className={classes.root}>
-      <rootProps.components.BaseFormControl
+      <FilterFormDeleteIcon
         variant="standard"
-        sx={{ flexShrink: 0, justifyContent: 'flex-end', marginRight: 0.5, marginBottom: 0.2 }}
-        {...rootProps.componentsProps?.baseFormControl}
+        {...baseFormControlProps}
+        {...deleteIconProps}
+        sx={deleteIconProps?.sx || {}}
```

Comment:
Why are you setting `sx` here? In the line above you also spread all props, including `sx`.

Reply:
Yes, it's useless

I only let one `sx` for link operators input, because the `hasLinkOperatorColumn` and `showMultiFilterOperators` are easier to read in the render than in the styled function
```jsx
sx={{
	  display: hasLinkOperatorColumn ? 'block' : 'none',
	  visibility: showMultiFilterOperators ? 'visible' : 'hidden',
	  ...(baseFormControlProps.sx || {}),
	  ...(linkOperatorInputProps.sx || {}),
}}
```